### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by billdenney**
This adds a docker file to easily work with the development version.

### CODE REVIEW

File: research/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+       sudo tzdata \
+    && DEBIAN_FRONTEND=noninteractive \
+       ln -fs /usr/share/zoneinfo/UTC /etc/localtime \
+    && DEBIAN_FRONTEND=noninteractive \
+       dpkg-reconfigure --frontend noninteractive tzdata
+
+
+RUN git clone https://github.com/ankitrohatgi/WebPlotDigitizer.git \
+    && cd WebPlotDigitizer \
+    && sed s/wine// setupUbuntuDev.sh > setupUbuntuDev-nowine.sh \
```
Comment (ankitrohatgi) [inline research/Dockerfile:26]: This is probably going to break now since I have changed `wine` to `wine-stable`.

Comment (billdenney) [reply research/Dockerfile]: Good catch!

**Comment by ankitrohatgi** [comment]
Looks like this is ready to merge? Sorry I missed that you had addressed all the comments weeks ago :(

**Comment by billdenney** [comment]
Thanks!

**Pull request merged**